.mailmap

```diff
@@ -27,3 +27,8 @@ Trăistaru Andrei Cristian <atc@amazon.com>
 Trăistaru Andrei Cristian <atc@amazon.com> <56828222+andreitraistaru@users.noreply.github.com>
 Sudan Landge <sudanl@amazon.com> <sudanl@amazon.co.uk>
 Takahiro Itazuri <itazur@amazon.com> <zulinx86@gmail.com>
+Jack Thomson <jackabt@amazon.com> <jackabt@amazon.co.uk>
+Ashwin Ginoria <aginoria@amazon.com> <ec2-user@ip-10-0-10-88.us-west-2.compute.internal>
+Muskaan Singla <msinglaa@amazon.com> <ec2-user@ip-172-31-1-28.us-west-2.compute.internal>
+Egor Lazarchuk <yegorlz@amazon.co.uk>
+Sudan Landge <sudanl@amazon.com> <119602619+sudanl0@users.noreply.github.com>
```

Comment:
nit: could we group Sudan's entries together? I think there are 3, but I am not sure the 1st one is needed.

Reply:
fixed